core/trino-main/src/main/java/io/trino/operator/DriverContext.java

```diff
@@ -275,9 +275,11 @@ public CounterStat getOutputPositions()
 
     public long getPhysicalWrittenDataSize()
     {
-        return operatorContexts.stream()
-                .mapToLong(OperatorContext::getPhysicalWrittenDataSize)
-                .sum();
+        long physicalWrittenBytes = 0;
+        for (OperatorContext context : operatorContexts) {
```

Comment:
Please add comments to these functions that we are avoiding streams for performance reasons so that somebody else doesn't try to "improve" the code later by bringing back stream